src/assets/scripts/client/base/ModelSource/modelSourceClassMap.js

```diff
@@ -17,7 +18,15 @@ export const CLASS_MAP = {
      * @type {FixModel}
      * @final
      */
-    FixModel: FixModel
+    FixModel: FixModel,
```

Comment:
just read through the `ModelSourceFactory` and `ModelSourcePool` for more info on this stuff.  It's really just a collection of class instances that are removed from the collection when in use and can be returned to the collection when not in use.

this is the first time I've build an object pool and I think it turned out ok.  I'd love to use it with more stuff eventually, but for now it's just the two constructors.